DSharpPlus.Commands/Processors/SlashCommands/Translation/Locales.cs

```diff
@@ -0,0 +1,39 @@
+namespace DSharpPlus.Commands.Processors.SlashCommands.Translation;
+
+using System.Diagnostics.CodeAnalysis;
+
+[SuppressMessage("Roslyn", "CA1707", Justification = "Underscores are required to be compliant with Discord's API.")]
+public enum Locales
```

Comment:
the dotnet naming convention requests that only flags enums are set in plural, whereas this enum should be Locale